src/common/Storage.tsx

```diff
@@ -0,0 +1,59 @@
+import moment from 'moment';
+import { AccountTokens } from "./types/Addresses";
+
+export function storeTokens(tokens: AccountTokens) {
+    const storage = getStorage();
+    clearTokens();
+    for(const address of Object.keys(tokens)) {
+        storage.setItem(TOKEN_KEY_PREFIX + address, JSON.stringify(tokens[address]));
+    }
+}
+
+function getStorage(): Storage {
+    return window.localStorage;
+}
+
+export function clearTokens() {
+    const toRemove: string[] = [];
+    forEachStorageKey(key => {
+        if(key!.startsWith(TOKEN_KEY_PREFIX)) {
+            toRemove.push(key!);
+        }
+    });
+
+    const storage = getStorage();
+    toRemove.forEach(key => storage.removeItem(key));
+}
+
+function forEachStorageKey(callback: (key: string) => void) {
+    const storage = getStorage();
+    for(let i = 0; i < storage.length; ++i) {
+        const key = storage.key(i);
+        callback(key!);
+    }
+}
```

Comment:
I would rather filter on prefix inside function `forEachStorageKey`

Reply:
The goal is to reuse this in other cases than tokens, that's why filtering is done in the callback.

Reply:
But the prefix can be given as parameter to the method. Anyway, it's a detail...